assets/stylesheets/common.css

```
@@ -282,8 +291,126 @@ footer > div:first-child > ul {
-.error {
-  color: #ff0000;
-  text-align: center;
+#alert-card {
+  display: flex;
+  align-items: center;
+  padding: 20px;
+  border-radius: 10px;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+  max-width: 400px;
+  width: 35%;
+  role: "alert";
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)